Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.20 as builder
```

Comment:
I'd love to add a dockerfile for this repo! Can you open a second PR for that? (The docker / publish workflow change can go in as-is I think.)